services/debugger/transformation/transformationStatusUploader.go

```diff
@@ -243,10 +243,30 @@ func (h *Handle) UploadTransformationStatus(tStatus *TransformationStatusT) bool
 			h.processRecordTransformationStatus(tStatus, transformation.ID)
 		} else {
 			tStatusUpdated := *tStatus
-			lo.Slice(tStatusUpdated.UserTransformedEvents, 0, h.limitEventsInMemory+1)
 			tStatusUpdated.Destination.Transformations = []backendconfig.TransformationT{transformation}
 			tStatusUpdated.UserTransformedEvents = lo.Slice(tStatusUpdated.UserTransformedEvents, 0, h.limitEventsInMemory+1)
 			tStatusUpdated.FailedEvents = lo.Slice(tStatusUpdated.FailedEvents, 0, h.limitEventsInMemory+1)
+			// while caching, we may only store the messageIDs of the events in the cache
+			// done to avoid displaying events as dropped
```

Comment:
Is this comment necessary? We just need to keep the struct consistent after trimming down the number of events it holds

Reply:
👍